types/react-slider/index.d.ts

```diff
@@ -1,12 +1,12 @@
-// Type definitions for react-slider 1.0
+// Type definitions for react-slider 1.1
 // Project: https://github.com/zillow/react-slider
 // Definitions by: Jason Unger <https://github.com/jsonunger>
 //                 Björgvin Bæhrenz Þórðarson <https://github.com/bjorgvin>
+//                 Loïc Huder <https://github.com/loichuder>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
 import * as React from 'react';
-import * as motion from 'react-motion';
```

Comment:
Not sure what this was doing here

Reply:
seems to be there from the very beginning of time:
https://github.com/loichuder/DefinitelyTyped/commit/917cb109d0cc654588357ccf3171648d3e10bcd1
current package has no dependency to that one, let's drop